Restore.cmd

```diff
@@ -1,3 +1,3 @@
-@if not defined EchoOn @echo off
-powershell -noprofile -executionPolicy RemoteSigned -file "%~dp0\build\scripts\restore-legacy.ps1" %*
+@echo off
```

Comment:
I would agree if we had any logic here.  But there is just one line here and echo has no value in debugging it.  It also doesn't pass on its value to Powershell (which is **really** unfortunate) otherwise I'd be hugely in favor of doing that.